source/api.rst

```diff
@@ -0,0 +1,613 @@
+PXC API Documentation
+=====================
+
+.. raw:: html
+
+   <style>
+   
+   .toggle {
+        background: none repeat scroll 0 0 #ffebcc;
+        padding: 12px;
+        line-height: 24px;
+        margin-bottom: 24px;
+    }
+   
+   .toggle .header {
+       display: block;
+       clear: both;
+       cursor: pointer;
+   }
+   
+   .toggle .header:after {
+       content: " ▶";
+   }
+   
+   .toggle .header.open:after {
+       content: " ▼";
+   }
+   </style>
+
+Percona Operator for Percona XtraDB Cluster provides an `aggregation-layer extension for the Kubernetes API <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/>`_. Please refer to the
+`official Kubernetes API documentation <https://kubernetes.io/docs/reference/>`_ on the API access and usage details.
+The following subsections describe the Percona XtraDB Cluster API provided by the Operator.
+
+.. contents:: :local:
+
+Prerequisites
+-------------
+
+1. prepare
+
+   .. code-block:: yaml
+
+      # set correct API adress
+      KUBE_CLUSTER=$(kubectl config view --minify -o jsonpath='{.clusters[0].name}')
+      API_SERVER=$(kubectl config view -o jsonpath="{.clusters[?(@.name==\"$KUBE_CLUSTER\")].cluster.server}")
+
```

Comment:
As I can see for now we work with default namespace but we need to show how to work with different namespaces . So I suggest adding the following:
```
NAMESPACE='pxc'
kubectl create namespace $NAMESPACE
```